belvedere/src/main/java/zendesk/belvedere/ImageStream.java

```diff
@@ -74,8 +77,12 @@ public void success(List<MediaResult> result) {
                 }
 
                 notifyImageSelected(filteredMediaResult);
+
+                cancel();
```

Comment:
Do we have to call `cancel` here since we are already executing `success`?

Reply:
I'd be interested about the reasoning here, do we have issues with this callback firing multiple times?

Reply:
Well this is embarrassing. This was a leftover from one of my earlier approaches to a solution, I didn't realise the change was still there. Will revert.